content/en/seax/seax_messaging/seax-widget.md

```diff
@@ -0,0 +1,200 @@
+---
+title: "SeaX Widget"
+description: "The SeaX Widget allows you to integrate a widget button that your customers can click on and directly send a message to your SeaX Messaging account."
+date: 2024-04-25T08:48:57+00:00
+lastmod: 2024-04-25T08:48:57+00:00
+draft: false
+images: []
+menu:
+  seax:
+    parent: "seax_enterprise"
+aliases:
+   - /seax/seax_intro/01-introduction/
+   - /seax/demo_videos/videos/
+weight: 102
+toc: true
+---
+
+## Overview
+In this article, we will introduce the SeaX Widget, a feature of the SeaX Messaging platform that allows you to integrate a widget button that your customers can click on and directly send a message to your SeaX Messaging account. You will learn how you can set up the SeaX Widget and integrate it into your website or app.
```

Comment:
Please call this product SeaX Messaging instead of SeaX whenever you refer to the product. I've updated all references in this article.

Reply:
thanks!